psi4/src/psi4/detci/params.cc

```diff
@@ -74,7 +74,7 @@ void CIWavefunction::get_parameters(Options &options) {
     Parameters_->wfn = options.get_str("WFN");
 
     // CDS-TODO: Check these
-    print_ = 1;
+    Parameters_->print_ = 1;
```

Comment:
Why is this needed? I would _expect_ that `CIWavefunction` is a wavefunction subclass and thus inherits the wavefunction's `print_` field. It doesn't need additional "parameters".

Reply:
I added `print_` to the parameters struct so that it can be passed to `CIVect`, where the un-set `print_lvl` used to live. `CIVect` gets passed the parameters struct, but not the wavefunction.